app/code/core/Mage/CatalogInventory/Model/Stock.php

```diff
@@ -159,8 +174,18 @@ public function registerProductsSale($items)
      */
     public function revertProductsSale($items)
     {
+        Mage::dispatchEvent(self::EVENT_CORRECT_STOCK_ITEMS_QTY_BEFORE, array(
+        'stock'     => $this,
+        'items_obj' => (object)array('items' => &$items),
+        'operator'  => '-'
```

Comment:
This also seems to be copy/paste error, why doesn't the operator match the one passed to correctItemsQty?